### ISSUE TITLE
Add option to save current file before opening in editor

### ISSUE THREAD

**Issue by W4RH4WK** [generalov/open-in-editor-vscode#5]
I am missing this feature. Often I invoke the external editor using this extension without saving the file. Of course the editor will display the old version.

I'd like to have a configurable option which enables autosaving just before the file gets opened in the editor.

**Comment by generalov** [comment]
@W4RH4WK, thank you so much for the idea. I'll try to look into that next weekends.

**Comment by huyz** [comment]
@W4RH4WK have you considered setting saving `onFocusChange`? https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save

**Comment by W4RH4WK** [comment]
No; but that might work. However, this issue is over 4 years old; not sure whether this is still relevant for me as my workflow has changed a lot.

**Comment by huyz** [comment]
Yep, more for other people who may just run into this thread.
